ark/utils/io_utils.py

```diff
@@ -36,19 +36,20 @@ def validate_paths(paths):
                     f'and to reference as \'../data/path_to_data/myfile.tif\'')
 
 
-def list_files(dir_name, substrs=None):
+def list_files(dir_name, substrs=None, is_extensions=True):
     """ List all files in a directory containing at least one given substring
 
     Args:
         dir_name (str):
             Parent directory for files of interest
         substrs (str or list):
             Substring matching criteria, defaults to None (all files)
+        is_extensions (bool):
+            If True, will match on file extensions, if False, will match exact file names
```

Comment:
This isn't exactly correct. It won't just match extensions if is_extensions is true. That's why I'm not sure I like the name_change for this flag

Reply:
Changed to `exact_match`, which I think is more informative.